math/int_test.go

```diff
@@ -43,6 +43,19 @@ func (s *intTestSuite) TestFromUint64() {
 	}
 }
 
+func (s intTestSuite) TestNewIntFromBigInt() {
```

Comment:
```suggestion
func (s (*intTestSuite) TestNewIntFromBigInt() {
```

For making the linter happy.